### PR TITLE
Add Premium packs to the "Successfully Uploaded Packs" slack message

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/31483

## Description
I added to the upload packs flow the ability to recognize all private packs that were updated successfully .
The flow knows which private pack was updated by comparing `contentCommitHash` field between public index json and the pack's metadata.

## Screenshots
Screen Shot 2021-01-12 at 13 47 58

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Tests/Marketplace/upload_packs.py [hunk 1]
```diff
@@ -464,14 +465,51 @@ def update_index_with_priced_packs(private_storage_bucket: Any, extract_destinat
...
+        index_folder_path (str): The public index folder path.
+
+    Returns:
+        updated_private_packs (list) : a list of all private packs id's that were updated.
+
+    """
+    updated_private_packs = []
+
+    with open(os.path.join(index_folder_path, f"{GCPConfig.INDEX_NAME}.json")) as public_index_file:
+        public_index_json = json.load(public_index_file)
```
Comment (guyfreund) [inline Tests/Marketplace/upload_packs.py:497]: I suggest using `load_json` from `marketplace_services` as it has some error handling

File: Tests/Marketplace/upload_packs.py [hunk 2]
```diff
@@ -464,14 +465,51 @@ def update_index_with_priced_packs(private_storage_bucket: Any, extract_destinat
...
+    public_packs = public_index_json.get("packs", {})
```
Comment (guyfreund) [inline Tests/Marketplace/upload_packs.py:498]:
```suggestion
    private_packs_from_public_index = public_index_json.get("packs", {})
```
naming here is a bit confusing, let's replace it with something more informative, here's a suggestion

File: Tests/Marketplace/upload_packs.py [hunk 3]
```diff
@@ -464,14 +465,51 @@ def update_index_with_priced_packs(private_storage_bucket: Any, extract_destinat
...
+
+    for pack in private_packs:
+        private_pack_id = pack.get('id')
+        new_private_commit_hash = pack.get('contentCommitHash', "")
```
Comment (guyfreund) [inline Tests/Marketplace/upload_packs.py:502]: is this the commit hash from content repo or from the pack's private repo?

File: Tests/Marketplace/upload_packs.py [hunk 4]
```diff
@@ -794,11 +835,12 @@ def main():
     # google cloud bigquery client initialized
     bq_client = init_bigquery_client(service_account)
     packs_statistic_df = get_packs_statistics_dataframe(bq_client)
+    updated_private_packs = []
```
Comment (guyfreund) [inline Tests/Marketplace/upload_packs.py:838]:
```suggestion
    updated_private_packs_ids = []
```
I think its better to change the name here, as the two other pack lists being used in the main function are of type `Pack` or dict

**Review by guyfreund — CHANGES_REQUESTED** [review]
Very Nice! added few minor changes

File: Tests/Marketplace/upload_packs.py (hunk 1 above)
Comment (abaumgarten) [reply Tests/Marketplace/upload_packs.py]: Done

File: Tests/Marketplace/upload_packs.py (hunk 2 above)
Comment (abaumgarten) [reply Tests/Marketplace/upload_packs.py]: Done

File: Tests/Marketplace/upload_packs.py (hunk 3 above)
Comment (abaumgarten) [reply Tests/Marketplace/upload_packs.py]: `new_private_commit_hash` was the commitHash from pack_metadata file. I changed both variables names (old and new) into better ones

File: Tests/Marketplace/upload_packs.py (hunk 4 above)
Comment (abaumgarten) [reply Tests/Marketplace/upload_packs.py]: Done

**Comment by abaumgarten** [comment]
@guyfreund 
I ran a successful upload-flow build (after changes according to CR), can see here:
https://app.circleci.com/pipelines/github/demisto/content/60619/workflows/f861b8bc-cdbd-4e53-8276-267a26a5c7f7/jobs/252114
https://app.circleci.com/pipelines/github/demisto/content/60619/workflows/f861b8bc-cdbd-4e53-8276-267a26a5c7f7/jobs/252115
https://app.circleci.com/pipelines/github/demisto/content/60619/workflows/f861b8bc-cdbd-4e53-8276-267a26a5c7f7/jobs/252134

slack message: 
Screen Shot 2021-01-14 at 12 55 03

**Review by guyfreund — APPROVED** [review]
👏

**Review by guykeller — APPROVED** [review]

**Pull request merged**